source/standards/source-code/working-with-git.html.md.erb

```diff
@@ -0,0 +1,196 @@
+---
+title: Working with Git
+last_reviewed_on: 2022-03-21
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+## Default branch name
+
+Your repository's default branch should be called `main`.
+
+To configure git to use `main` for new repositories, first make sure you are using git 2.28 or later, then run:
+
+    git config --global init.defaultBranch main
+
+To migrate existing repositories from `master` to `main`, you can use
+[GitHub's branch renaming
+tool](https://github.com/github/renaming#renaming-existing-branches).
+
+Before you rename your default branch, you should consider the impact this will have on:
+
+ - internal users
+ - external users
+ - your continuous integration (CI) system (for example, Concourse pipelines or GitHub actions)
+
+You can configure some CI systems to check out the default branch rather than hardcoding a branch name:
+
+ - [GitHub Actions](https://github.blog/changelog/2020-07-22-github-actions-better-support-for-alternative-default-branch-names/) supports a `$default-branch` macro
+ - [concourse-git-resource](https://github.com/concourse/git-resource): if you do not set `branch`, it will check out the default branch
+
+## Commits
+
+### Atomic commits
+
+Each commit should be atomic, meaning that each commit should only include changes that achieve a given, granular, step.
+They should also be logical steps that follow each other. As Anna Shipman describes it,
```

Comment:
Small suggestion - slight rewording to hopefully make it a bit clearer, and split it up over three bullet points to make it easier to scan:

```suggestion
Each commit should be atomic - this means that each commit:

* is self-contained
* only includes changes that achieve a specific step
* should be in a logical order

As Anna Shipman describes it,
```